presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -5,6 +5,28 @@ Tuning Presto
 The default Presto settings should work well for most workloads. The following
 information may help you if your cluster is facing a specific performance problem.
 
+
+When setting up a cluster for a specific workload it may be necessary to adjust the
+following properties to ensure optimal performance:
+
+  * :ref:`distributed-joins-enabled<general-properties>`
+  * :ref:`query.max-memory<query-properties>`
+  * :ref:`query.max-memory-per-node<query-properties>`
+  * :ref:`query.initial-hash-partitions<query-properties>`
+  * :ref:`task.concurrency<task-properties>`
+
+Those and other Presto properties are described in :doc:`properties article<properties>`.
+
+As an example, on a 11-node cluster dedicated to Presto (1 Coordinator + 10 Workers) with 8-core CPU and 128GB of RAM per node, you might want to start tuning with following values:
+
+  * `query.max-memory = 200GB`
+  * `query.max-memory-per-node = 32GB`
+  * `query.initial-hash-partitions = 10`
+  * `task.concurrency = 8`
```

Comment:
maybe some explanation how it was calculated would be beneficial to user as well. So she/he could calculate these numbers in her/his case.